CONTRIBUTING.md

```diff
@@ -1,3 +1,13 @@
+### Project License:  Eclipse Public License v1.0
+
+- You will only Submit Contributions where You have authored 100% of the content.
+- You will only Submit Contributions to which You have the necessary rights. This means that if You are employed You have received the necessary permissions from Your employer to make the Contributions.
```

Comment:
What specifically does this mean? Note that almost all US companies assert that any code written by the employer--even code written on their own time, on their personal machines--are copyrighted by the company.